test/blackbox/CMakeLists.txt

```diff
@@ -205,9 +205,14 @@ endif()
 
 # OpenSSL on Windows requires a hint on which config file to load
 if(WIN32 AND OPENSSL_FOUND)
-    get_filename_component(OPENSSL_DIR "${OPENSSL_INCLUDE_DIR}" DIRECTORY)
-    set(OPENSSL_CONF "${OPENSSL_DIR}/bin/cnf/openssl.cnf")
-    unset(OPENSSL_DIR)
+
+    find_package(LibP11)
+
+    if(LibP11_FOUND)
```

Comment:
Why is this condition necessary?
Do we need to set OPENSSL_CONF if LibP11 is not found?

Reply:
PKCS11 testing requires such a complex configuration and dependency settings (on windows at least) that will (probably) only suit our CI.
I'm going to subordinate the windows configuration file settings to the presence of the `EPROSIMA_BUILD`  flag to prevent an override of user-specific Open-SSL settings.
Although, is quite probable that the PKCS11 testing will fail on a user machine unless [this settings](https://fast-dds.docs.eprosima.com/en/v2.5.0/installation/sources/sources_windows.html?highlight=softhsm#libp11-and-softhsm-libraries) are enforced.